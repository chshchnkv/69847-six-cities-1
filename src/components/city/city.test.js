import React from "react";
import renderer from "react-test-renderer";
import City from "./city";

const mockCity = {
  id: 1,
  name: `Amsterdam`,
  location: {
    longitude: 52.38333,
    latitude: 4.9,
    zoom: 12,
  }
};

it(`City renders correctly`, () => {
  const onActivateCity = jest.fn();
  const tree = renderer.create(<City longitude={mockCity.location.longitude} latitude={mockCity.location.latitude} title={mockCity.name} id={mockCity.id} isActive={true} onActivateCity={onActivateCity}/>);
  expect(tree).toMatchSnapshot();
});
