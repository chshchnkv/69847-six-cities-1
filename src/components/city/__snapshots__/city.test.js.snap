// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`City renders correctly 1`] = `
<a
  className="locations__item-link tabs__item tabs__item--active"
  href="#"
  onClick={[Function]}
>
  <span>
    Amsterdam
  </span>
</a>
`;
